src/scripts/web-standalone-template.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <title>My App</title>
+      <head>
+        <script>
+          window.process = {
+          stdout: {write: function(s) { console.log(s); } },
+          stderr: {write: function(s) { console.error(s); } },
+          argv: [],
+          exit: function(err) {
+          if (err != 0) {
+          throw new Error("Program ending with signal: ", err);
+          }
+          }
+          }
+
+        </script>
+      </head>
+
+    <body>
+        <h1>My App</h1>
+    </body>
+    <script type="text/javascript" src="{{{compiled-jarr-file}}}"></script>
```

Comment:
In `make-standalone`, we replace `{{{compiled-jarr-file}}}` with the actual name of the file.